features_and_tools/custom_operations/convertoperation.rst

```diff
@@ -33,21 +37,4 @@ Configuration
          },
          ...
       ]
-   }
-
-Release notes
--------------
-
-Version 0.2.0
-^^^^^^^^^^^^^
```

Comment:
You did not remove this for BinaryWrapper. Why the difference?

Reply:
Removed it there as well